.travis.yml

```diff
@@ -1,11 +1,19 @@
 sudo: required
 
 install:
-    - wget 'https://sourceforge.net/projects/astyle/files/latest/download'
-    - tar -zxvf download
-    - (cd astyle/build/gcc && make release && sudo make install)
-    - rm astyle -rf
+ - sudo Installer/MainInstaller.sh
+ - wget 'https://sourceforge.net/projects/astyle/files/latest/download'
+ - tar -zxvf download
+ - (cd astyle/build/gcc && make release && sudo make install)
+ - rm astyle -rf
 
 script:
-    - "! (astyle *.h *.cpp *.hpp -r --dry-run --options=astylerc | grep Formatted)"
+ - make
+ - "! (astyle *.h *.cpp *.hpp -r --dry-run --options=astylerc | grep Formatted)"
+
```

Comment:
Fix the white spacing, there's extra at the bottom and the points aren't indented.